app/services/post_bookmarkable.rb

```diff
@@ -80,4 +80,19 @@ def self.after_create(guardian, bookmark, opts)
   def self.after_destroy(guardian, bookmark, opts)
     sync_topic_user_bookmarked(guardian.user, bookmark.bookmarkable.topic, opts)
   end
+
+  def self.cleanup_deleted
```

Comment:
This and the `TopicBookmarkable#cleanup_deleted` methods are tested via the https://github.com/discourse/discourse/pull/17202/files#diff-951c6604b881ff0d66025347355d9c94e62cce171f9862d40feb024f5f970028 bookmark spec, if necessary I can move them out into the relevant bookmarkable specs?

Also not quite sure about the `cleanup_deleted` name, any suggestions for naming are welcome.

Reply:
`cleanup_deleted` looks fine to me 👍 

Other ideas: `delete_trashed`, `delete_deleted` 😅, `purge_deleted`